conf/php.ini

```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
```

Comment:
Yes, I believe that file is only for standard php -- not php-fpm. There's not much going on in that php.ini file that would make any difference anyways. 64MB memory also seems really puny 😷